src/sources/socket/udp.rs

```diff
@@ -128,7 +127,7 @@ impl UdpConfig {
             host_key: default_host_key(),
             port_key: default_port_key(),
             receive_buffer_bytes: None,
-            framing: default_framing_message_based(),
+            framing: None,
```

Comment:
the default is now calculated when initializing the socket, as done with the `TcpConfig` and `UnixConfig`